package.json

```diff
@@ -27,6 +27,7 @@
     "@babel/plugin-transform-parameters": "^7.1.0",
     "@babel/plugin-transform-template-literals": "^7.0.0",
     "@babel/preset-env": "^7.1.0",
+    "@mapbox/mapbox-gl-rtl-text": "github:mapbox/mapbox-gl-rtl-text",
```

Comment:
Any reason for not using the latest npm release directly ?  
https://www.npmjs.com/package/@mapbox/mapbox-gl-rtl-text/v/0.2.1

Reply:
This is the changes made by "npm install" command !